examples/fabric/cloud_router/go/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 toolchain go1.22.6
 
 require (
-	github.com/equinix/pulumi-equinix/sdk 0.14.0+dirty
+	github.com/equinix/pulumi-equinix/sdk 0.14.0
```

Comment:
This will come back in the next release.  There's something missing in the release process that is causing it to have a dirty work tree, so we'll need to find & fix that in order to fix this.